telegram/ext/messagequeue.py

```diff
@@ -283,40 +268,36 @@ def __call__(self, promise, is_group_msg=False):
 
 
 def queuedmessage(method):
-    '''A decorator to be used with `telegram.bot.Bot` send* methods.
+    """
+    A decorator to be used with :attr:`telegram.Bot` send* methods.
 
     Note:
-        As it probably wouldn't be a good idea to make this decorator a
-        property, it had been coded as decorator function, so it implies that
-        **first positional argument to wrapped MUST be self**\.
+        As it probably wouldn't be a good idea to make this decorator a property, it has been coded
+        as decorator function, so it implies that first positional argument to wrapped MUST be
+        self.
 
     The next object attributes are used by decorator:
 
     Attributes:
-        self._is_messages_queued_default (:obj:`bool`): Value to provide
-            class-defaults to `queued` kwarg if not provided during wrapped
-            method call.
-        self._msg_queue (:obj:`telegram.ext.messagequeue.MessageQueue`):
-            The actual `MessageQueue` used to delay outbond messages according
-            to specified time-limits.
+        self._is_messages_queued_default (:obj:`bool`): Value to provide class-defaults to
```

Comment:
The docs are written by @thodnev . Since I don';t understand shit about the messagequeue I only prettyfied them. The contents is his responsibility.